docs/user-guide/stack_parameters_override.md

```diff
@@ -1,22 +0,0 @@
-# Stack Parameter overrides
```

Comment:
I think there's some wisdom in here that we're losing in the new documentation. Specifically this contains examples and warnings about developers stomping on each other's settings if they are not in source control.